Source/WebKit/NetworkProcess/storage/OriginStorageManager.h

```diff
@@ -65,6 +65,8 @@ class OriginStorageManager {
     OptionSet<WebsiteDataType> fetchDataTypesInList(OptionSet<WebsiteDataType>);
     void deleteData(OptionSet<WebsiteDataType>, WallTime);
     void moveData(OptionSet<WebsiteDataType>, const String& localStoragePath, const String& idbStoragePath);
+    bool didWriteOriginToFile() const { return m_didWriteOriginToFile; }
+    void setDidWriteOriginToFile(bool didWriteOriginToFile) { m_didWriteOriginToFile = didWriteOriginToFile; }
```

Comment:
This only ever sets it to true so I think this function shouldn't have a parameter:
`void setDidWriteOriginToFile() { m_didWriteOriginToFile = didWriteOriginToFile; }`
or
`void markDidWriteOriginToFile() { m_didWriteOriginToFile = didWriteOriginToFile; }`

Reply:
Sure.